test/models/deploy_service_test.rb

```diff
@@ -160,6 +160,15 @@ class DeployServiceTest < ActiveSupport::TestCase
       job_execution.send(:run!)
     end
 
+    it "sends Zendesk notifications if the stage has them enabled" do
+      stage.stubs(:comment_on_zendesk_tickets?).returns(true)
+
+      ZendeskNotification.any_instance.expects(:deliver)
```

Comment:
we could assert that it responds_to deliver, but if someone renames the method again it will break [this](https://github.com/zendesk/samson/blob/master/test/models/zendesk_notification_test.rb#L20) test